test/integration/shimmer/module-load-fixture-errored.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+module.exports = () => {
```

Comment:
I created a separate module to avoid hitting the require cache. Typically, I use mockery to reset the require cache, but this is an alternate solution.